src/viam/media/video.py

```diff
@@ -105,3 +106,37 @@ def is_supported(cls, mime_type: str) -> bool:
         if mime_type == cls.UNSUPPORTED:
             return False
         return mime_type in set(item.value for item in cls)
+
+    @classmethod
+    def from_proto(cls, format: Format.ValueType) -> Self:
+        """Returns the mimetype from a proto enum.
+
+        Args:
+            format (Format.ValueType): The mimetype in a proto enum.
+
+        Returns:
+            Self: The mimetype.
+        """
+        mimetypes = {
+            Format.FORMAT_RAW_RGBA: CameraMimeType.VIAM_RGBA,
+            Format.FORMAT_JPEG: CameraMimeType.JPEG,
+            Format.FORMAT_PNG: CameraMimeType.PNG,
+            Format.FORMAT_RAW_DEPTH: CameraMimeType.PCD,
```

Comment:
FORMAT_RAW_DEPTH is actually the MIME type "image/vnd.viam.dep", it is not the PCD format "pointcloud/pcd".

Reply:
https://github.com/viamrobotics/rdk/blob/main/utils/mime.go#L19

Reply:
updated